src/Database/Database.php

```diff
@@ -1937,4 +1947,127 @@ public static function convertQueries(Document $collection, array $queries): arr
         }
         return $queries;
     }
+
+
+    /**
+     * Increase Collection Attribute by a value
+     *
+     * @param string $collection
+     * @param string $id
+     * @param string $attribute
+     * @param int|float $value
+     * @param null $max
+     * @return bool
+     *
+     * @throws AuthorizationException
+     * @throws Exception
+     */
+    public function increaseDocumentAttribute(string $collection, string $id, string $attribute, int|float $value = 1, $max = null): bool
```

Comment:
```suggestion
    public function increaseDocumentAttribute(string $collection, string $id, string $attribute, int|float $value = 1, ?int $max = null): bool
```

shouldn't max be a nullable int ?

Reply:
yes and added a float option as well